client/src/main/java/com/netflix/conductor/client/task/WorkflowTaskCoordinator.java

```diff
@@ -285,8 +281,20 @@ private void pollForTask(Worker worker) {
 			List<Task> tasks = client.poll(taskType, worker.getIdentity(), worker.getPollCount(), worker.getLongPollTimeoutInMS());
 			sw.stop();
 			logger.debug("Polled {} and receivd {} tasks", worker.getTaskDefName(), tasks.size());
+
 			for(Task task : tasks) {
-				es.submit(()->execute(worker, task));	
+				ListenableFuture<TaskResult> future = es.submit(() -> execute(worker, task));
```

Comment:
Since you're not doing anything with the future, this isn't any different than

```
es.submit(() -> {
    try {
        execute(worker, task);
    } catch (Exception e) {
        task.setStatus(Task.Status.FAILED);
        TaskResult result = new TaskResult(task);
        handleException(t, result, worker, true, task);
    }
});
```